src/search-box/__tests__/search-box.spec.ts

```diff
@@ -63,6 +63,26 @@ describe('SearchBox', () => {
         expect(refine).toHaveBeenCalledTimes(1);
         expect(refine).toHaveBeenCalledWith('the query');
       });
+
+      it('should not update query when out of sync and input is focused', () => {
+        const fixture = createRenderer({
+          defaultState,
+          template: '<ais-search-box></ais-search-box>',
+          TestedWidget: NgAisSearchBox,
+        })();
+        const widget = fixture.componentInstance.testedWidget;
+
+        const searchBox = widget.searchBox.nativeElement;
+        searchBox.focus();
+        searchBox.value = 'hello';
+        searchBox.dispatchEvent(new Event('input'));
+        fixture.detectChanges();
+        expect(searchBox.value).toEqual('hello');
+
+        widget.state.query = 'hel';
+        fixture.detectChanges();
+        expect(searchBox.value).toEqual('hello');
+      });
```

Comment:
Should we then blur the input and make sure that it does update?

Reply:
Added in 8f04646.